app/Http/Controllers/TranslationController.php

```diff
@@ -26,8 +30,9 @@ public function __construct(WordService $wordService, SignService $signService,
     public function createTranslation(Request $request)
     {
         $user = $this->userService->getUser();
+        $literal = $request->input('literal');
```

Comment:
Der er kun en `$args['literal']` fra endpoint. Der er ingen `$request->input('literal')`... Medmindre du mimer min sortInput og laver en FormRequest fra bunden med form validation - så kan du lave en `$request = new WordRequest($args['literal'])` fra endpoint og give det videre hertil...

Reply:
Altså: `php artisan make:request WordRequest` - vupti!

Reply:
For at undgå scope creep, opretter jeg en issue som jeg vil arbejde videre på det.